README.md

```diff
@@ -47,6 +47,14 @@ Read more about [UI-based plugin installation](https://docs.netlify.com/configur
       package = "@netlify/plugin-nextjs"
     ```
 
+ You can also add context-specific properties and environment variables to your `netlify.toml`. Read more about [deploy contexts](https://docs.netlify.com/configure-builds/file-based-configuration/#deploy-contexts) in our docs. For example:
```

Comment:
Do you think we can fit a link to https://docs.netlify.com/cli/get-started/#run-builds-locally in here somewhere?
This is still 🚀 without it

Reply:
we have a link to cli/get-started right below the installation steps so i think that should be fine? can always add later tho :)